### DISCUSSION TITLE
My project on github pages is not showing on mobile browser (404 error)

### DISCUSSION THREAD

**Discussion by AaronCur** [community/community#23651] [Other Feature Feedback, Questions, & Ideas]
<p>Hi everyone! So it was my first time messing with github pages today. I made a small react app today to learn some stuff and tried host it on github pages. It works perfectly when i try access the site on my laptop. However when i try access it on mobile the site doesnt seem to exist? Have i got some sort of weird error or do i have to add something to my code to support mobile ? (the app isnt at all optomized for mobile i just wanted to see if i could see it on my phone).</p>
<p>Here is the link <a href="https://aaroncur.github.io/RedSky/" rel="nofollow noopener">https://aaroncur.github.io/RedSky/</a></p>
<p>I appreciate any advice :slight_smile:</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/aaroncur">@aaroncur</a>,</p>
<p>I just tried this on my iPhone in Chrome and Safari browsers and it came up just fine on both. Are you still experienicing this problem on your end? If so, can you share which browser you’re using on your phone? Thanks!</p>

**Comment by AaronCur** [comment]
<p>Hi Nadia!<br>
Thank you for testing. Thankfully it’s working now. Just took a while to appear on mobile :slight_smile:</p>

**Comment by ekawahyu** [comment]
<p><a class="mention" href="/aaroncur">@aaroncur</a> Just to let you know, it seems that your web is showing 404 on my iPhone today, just now</p>

**Comment by tayyabferozi** [comment]
<p><a class="mention" href="/aaroncur">@aaroncur</a> how were you able to make it through?</p>

**Reply by SeytkhanB** [reply]
It is really easy man!

**Comment by dspyk77** [comment]
I know this is an old post, if anyone ends up here with the same issue as I did, add the html file name on the end of the URL. In my case the desktop browser has no issue with https://dspyk77.github.io/amazon-savings/ but on mobile it looks like this https://dspyk77.github.io/amazon-savings/index.html

**Reply by pantchayan** [reply]
Hey, Thank you for this. Do you know why this happens? Is there any fix to this? as I want my web app to be responsive.

https://pantchayan.github.io/three-christmas-game/ (Opens up Desktop Site on mobile)
vs
https://pantchayan.github.io/three-christmas-game/index.html (Opens up Mobile Site on mobile)

**Comment by SeytkhanB** [comment]
I tested I works in todays day 👍🏼
Can somebody help me with this app: https://seytkhanb.github.io/new-speed-typing-game/
In laptop it works just fine but on phone it doesn't work!
Any suggestions will be appreciated!

**Comment by Rilindi12** [comment]
Can someone tell me why im having problem with mobile device but in desktop works fine , this is my repo ? https://rilindi12.github.io/rilind-shabani/

**Reply by zetachua** [reply]
How did u fix it it works on my movile

**Reply by W4FUL4san** [reply]
It works fine on my mobile.

**Comment by ghost** [comment]
Hee where is the solution

**Comment by AndreaRandazzo** [comment]
Hello everyone, I have almost the same problem with my repo, on my laptop work fine but on iphone fall to try the call to the server.

**Comment by RodFoyagan** [comment]
My issue is in desktop it work fine but in mobile phone, the display is scrambled. Any help please?

**Comment by takaursa** [comment]
I'm also having the same problem.
My website shows correctly on desktop but not working on iPhone.

**Reply by zetachua** [reply]
Hi i just fixed it, so the issue for me was i pushed to repository after i build and deploy the project to gh pages, then it didnt work afterwards. So rmb to finish pushing to repository and then rebuild and redeploy ghpages then ur website hosted on ghpages will work on mobile and desktop

most likely if it doesnt work on mobile means it didnt work on desktop too cuz like me i pushed to my gh repo after i build and deployed cuz i got a few changes i wanted to save
